appserver/tests/appserv-tests/devtests/security/build.xml

```diff
@@ -109,6 +110,15 @@
         <ant dir="realmPerApp" target="all"/>  
         <ant dir="sful" target="all"/>  
     </target>
+    <target name="soteria">
```

Comment:
No necessarily but this has tags related to generating report. This is helpful in testing if some bug in reporting as for reporting we use checked in <APS_HOME>/lib/reporter.jar. Report is generated only from parent tests.